<!-- test-analysis-auto -->
@claude 以下のテスト失敗を修正してください。

**失敗ジョブ**: Lint
**Workflow Run**: https://github.com/sy-hash/mind-echo/actions/runs/23055008486

```
Lint	Run swift-format lint	2026-03-13T14:17:30.6584500Z MindEcho/MindEcho/Services/ExportService.swift:5:1: error: [OrderedImports] sort import statements lexicographically
```

テストが通るようにコードを修正し、ビルドが通ることも確認してください。

--- a/MindEcho/MindEcho/Services/ExportService.swift
+++ b/MindEcho/MindEcho/Services/ExportService.swift
@@ -1,8 +1,8 @@
 import AVFoundation
+import CoreText
 import Foundation
 import MindEchoAudio
 import MindEchoCore
-import CoreText
 import UIKit
 
 struct ExportServiceImpl: Exporting {

